webrender/src/debug_render.rs

```diff
@@ -2,14 +2,17 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-use api::{ColorU, DeviceIntRect, DeviceUintSize, ImageFormat, TextureTarget};
+use api::{ColorU, ColorF, DeviceIntRect, DeviceUintSize, ImageFormat, TextureTarget};
 use debug_font_data;
-use device::{Device, Program, Texture, TextureSlot, VertexDescriptor};
-use device::{TextureFilter, VertexAttribute, VertexAttributeKind};
+use device::{Device, ProgramId, Texture, TextureSlot, /*VertexDescriptor*/};
```

Comment:
do we need those commented out entries left?

Reply:
No, we don't. I will remove them.